chapter05/dcox/ex5_2

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+sub print_ruler_line {
+    print $_ % 10 foreach ( 1 .. 60 );
```

Comment:
yeah, this looks much cooler. I'm curious to know what this will actually print out? so tempted to try...